package.json

```diff
@@ -100,32 +87,30 @@
     "rxjs": ">=7.0.0"
   },
   "peerDependencies": {
-    "@angular/core": ">=12.1.2",
     "@ngx-translate/core": ">=13.0.0",
-    "@ngx-translate/http-loader": ">=6.0.0",
-    "typescript": ">=4.3.5"
```

Comment:
need to put back typescript as peer dependency and also bump ngx-translate to latest version